Documentation/building/linux-instructions.md

```diff
@@ -63,7 +63,7 @@ For example, for Ubuntu, you do the following:
 ellismg@linux:~$ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
 ellismg@linux:~$ echo "deb http://download.mono-project.com/repo/debian wheezy main" | sudo tee /etc/apt/sources.list.d/mono-xamarin.list
 ellismg@linux:~$ sudo apt-get update
-ellismg@linux:~$ sudo apt-get install mono-devel
+ellismg@linux:~$ sudo apt-get install mono-devel referenceassemblies-pcl
```

Comment:
We no longer should need mono or the referenceassemblies-pcl stuff to build either CoreFX or CoreCLR so this can be removed.